### PR TITLE
proot: disable seccomp

### PR DESCRIPTION
**Pull request by Piraty**
otherwise it segfaults on newer kernels,
see: https://github.com/proot-me/proot/issues/106

Patch from Debian10

### CODE REVIEW

File: srcpkgs/proot/template [hunk 1]
```diff
@@ -1,30 +1,27 @@
 # Template file for 'proot'
 pkgname=proot
 version=5.1.0
-revision=5
-wrksrc="PRoot-${version}"
+revision=7
```
Comment (Vaelatern) [inline srcpkgs/proot/template]: Double revision bump?

Comment (Piraty) [reply srcpkgs/proot/template]: thanks

**Comment by Vaelatern** [comment]
One comment for a future commit (this one waited too long for me to be willing to tear it apart) is that perhaps do_build is completely redundant now.

**Comment by Piraty** [comment]
@Vaelatern thanks. fixed in cadff983868f31e06975a93f7f59d9816472b1e1

**Comment by Vaelatern** [comment]
Thanks @Piraty !

**Pull request merged**